sage/index.py

```diff
@@ -41,8 +41,8 @@ def main():
         return
 
     # Additionally validate embedder and vector store compatibility.
-    if args.embedding_provider == "openai" and args.vector_store_provider != "pinecone":
-        parser.error("When using OpenAI embedder, the vector store type must be Pinecone.")
+    if args.embedding_provider == "openai" and args.vector_store_provider not in ["pinecone","chroma","faiss", "milvus", "qdrant"]:
```

Comment:
Would you mind using a `VectorStoreProvider` enum like we did for [RerankerProvider](https://github.com/Storia-AI/sage/blob/aa5cfeb9981c2d24a127f287658b6e426e7f2dd4/sage/reranker.py#L14)? Then we can get its values via `[member.value for member in VectorStoreProvider]`